### ISSUE TITLE
Can't start

### ISSUE THREAD

**Issue by firedev** [mxmvshnvsk/i18n-unused#2]
```
$ i18n-unused  display-unused --locales-path "config/locale"
(node:95074) UnhandledPromiseRejectionWarning: Error: Locales path is required
    at initialize (/Users/pain/.asdf/installs/nodejs/14.17.1/.npm/lib/node_modules/i18n-unused/dist/i18n-unused.umd.js:39:13)
    at Command.displayUnusedTranslations (/Users/pain/.asdf/installs/nodejs/14.17.1/.npm/lib/node_modules/i18n-unused/dist/i18n-unused.umd.js:148:20)
    at Command.listener [as _actionHandler] (/Users/pain/.asdf/installs/nodejs/14.17.1/.npm/lib/node_modules/i18n-unused/node_modules/commander/lib/command.js:466:17)
    at /Users/pain/.asdf/installs/nodejs/14.17.1/.npm/lib/node_modules/i18n-unused/node_modules/commander/lib/command.js:1170:65
    at Command._chainOrCall (/Users/pain/.asdf/installs/nodejs/14.17.1/.npm/lib/node_modules/i18n-unused/node_modules/commander/lib/command.js:1088:12)
    at Command._parseCommand (/Users/pain/.asdf/installs/nodejs/14.17.1/.npm/lib/node_modules/i18n-unused/node_modules/commander/lib/command.js:1170:27)
    at Command._dispatchSubcommand (/Users/pain/.asdf/installs/nodejs/14.17.1/.npm/lib/node_modules/i18n-unused/node_modules/commander/lib/command.js:995:25)
    at Command._parseCommand (/Users/pain/.asdf/installs/nodejs/14.17.1/.npm/lib/node_modules/i18n-unused/node_modules/commander/lib/command.js:1136:19)
    at Command.parse (/Users/pain/.asdf/installs/nodejs/14.17.1/.npm/lib/node_modules/i18n-unused/node_modules/commander/lib/command.js:841:10)
    at Object.<anonymous> (/Users/pain/.asdf/installs/nodejs/14.17.1/.npm/lib/node_modules/i18n-unused/bin/i18n-unused.js:48:9)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:95074) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:95074) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by mxmvshnvsk** [comment]
@firedev. Hi. I'll fix inline options later, now try to use config in root folder.

**Comment by mxmvshnvsk** [comment]
Fixed in [v0.0.8](https://www.npmjs.com/package/i18n-unused/v/0.0.8)
